src/promptflow/promptflow/_sdk/_tracing.py

```diff
@@ -61,13 +62,22 @@ def _invoke_pf_svc() -> str:
     port = get_port_from_config(create_if_not_exists=True)
     port = str(port)
     cmd_args = ["start", "--port", port]
+    hint_stop_message = (
+        f"You can stop the Prompt flow Tracing Server with the following command: '\033[1m pf service "
+        f"stop\033[0m'\nAlternatively, if no requests are made within {PF_SERVICE_HOUR_TIMEOUT} "
+        f"hours, it will automatically stop if Prompt flow is installed via pip."
```

Comment:
no need to mention this: it will automatically stop if Prompt flow is installed via pip